### PR TITLE
Do not use backpressure in browsers. Fixes #56.

### LINKED ISSUE #56: After first writeOrBuffer() state.writing remains true all the time
This is a tricky problem and I cannot locate the bug. Somehow, after upgrading to `"websocket-stream": "^1.3.0"`, my client can write only once and then nothing happens.

After some debugging I found out that `state.writing` is always true after the first write operation. This in writeOrBuffer() of `./node_modules/websocket-stream/node_modules/duplexify/node_modules/readable-stream/lib/_stream_writable.js`

```
  if (state.writing || state.corked)
    state.buffer.push(new WriteReq(chunk, encoding, cb));
  else
    doWrite(stream, state, false, len, chunk, encoding, cb);
```

In other words, after the first write operation, everything else is pushed in the buffer and nothing happens. Any clues?

### PR DESCRIPTION
**Pull request by mcollina**
@binarykitchen and @georgzoeller please test.

Fixed #56.

### CODE REVIEW

File: stream.js [hunk 1]
```diff
@@ -33,10 +34,19 @@ function WebSocketStream(target, protocols) {
...
 
-  function socketWrite(chunk, enc, next) {
+  function socketWriteNode(chunk, enc, next) {
     socket.send(chunk, next)
   }
 
+  function socketWriteBrowser(chunk, enc, next) {
+    try {
+      socket.send(chunk)
+      next()
```
Comment (mafintosh) [inline stream.js]: we shouldn't try catch the callback

``` js
try {
  socket.send(chunk)
} catch (err) {
  return next(err)
}

next()
```

**Comment by binarykitchen** [comment]
Ok, tested @mcollina, with your changes #56 is solved. Go ahead, move `next()` to the catch block, then merge and publish it please!

**Comment by mcollina** [comment]
Released as 1.3.1.

**Comment by georgzoeller** [comment]
Fixes my case as well.

**Pull request merged**